src/app/SharpRaven/Data/JsonPacket.cs

```diff
@@ -123,7 +123,16 @@ private static object Merge(SentryEvent @event)
                 }
             }
             else
-                result = JObject.FromObject(extra);
+            {
+                try
+                {
+                    result = JObject.FromObject(extra);
+                }
+                catch (ArgumentException e)
+                {
+                    result = JObject.Parse(string.Format(@"{{""{0}"":""{1}""}}", extra.GetType(), extra));
```

Comment:
This seems a bit sub-optimal. Can't we instantiate a new `JObject` and give it a property somehow?